platform/security/certificate/security-certificate-generator/src/main/java/org/codice/ddf/security/certificate/generator/CertificateCommand.java

```diff
@@ -18,42 +18,83 @@
 import java.security.cert.X509Certificate;
 import java.util.Arrays;
 
+import javax.annotation.Nullable;
+
 import org.apache.commons.lang.Validate;
 import org.bouncycastle.asn1.x500.RDN;
 import org.bouncycastle.asn1.x500.style.BCStyle;
 
 import ddf.security.SecurityConstants;
 
 public class CertificateCommand {
-
     /**
      * Pass in a string to use as the common name of the certificate to be generated.
-     * Exception thrown if 0 arguments or more than 1 argument.
+     * Exception thrown if 0 arguments or more than 2 argument (or more than 4 when -san is used).
+     * <pre>
+     * Arguments to this program are: <code>[-cn &lt;cn&gt;|-dn &lt;dn&gt;] [-san &lt;tag:name,tag:name,...&gt;]</code>
```

Comment:
I know ... I am just used to doing it that way. I also thought that {@code} generates the block differently in the javadoc then <pre> though.